website/templates/util/render_nodes.mako

```diff
@@ -1,5 +1,6 @@
 % if len(nodes):
     <ul class="list-group ${'sortable' if sortable and 'write' in user['permissions'] else ''}">
+      <span id="componentScope">
```

Comment:
The node title shift to the right after the icons load asynchronously. Add a `class="scripted"` tag here so that the span isn't shown until the knockout binding is complete.